tvm_executor/src/transaction_executor.rs

```diff
@@ -298,7 +305,40 @@ pub trait TransactionExecutor {
             fee.add(due_payment)?;
             acc.set_due_payment(None);
         }
-
+        if (tr.now() < acc.last_paid() + STORAGE_FEE_COOLER_TIME) && (is_same_thread_id) {
```

Comment:
Is this logic right? We check storage fee only one month later from the last call, if contract is called regularly, it won't be charged at all?